doc/install.md

```diff
@@ -51,7 +51,7 @@ You can also build your own executables from source code.
 
 &ensp; 2\. Make sure you have the necessary libraries installed:
 
- * Linux (Ubuntu): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git curl`
+ * Linux (Ubuntu 18.04): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git curl`
```

Comment:
16.04 should also work (this is the version used in this repo's [CI](https://github.com/livepeer/go-livepeer/blob/d0ea014d68488c1bc26c4e5882949a0d1dc1c86c/docker/Dockerfile.build-linux#L1)).

Could we include 16.04 here as well?

Reply:
Thanks @yondonfu - added 16.04